validation/tests/test_bulk_recordings_search.py

```diff
@@ -0,0 +1,106 @@
+"""
+Unit/Integration tests for the bulk recordings search API.
+"""
+import urllib
+
+import pytest  # type: ignore
+from django.core.management import call_command
+from django.shortcuts import reverse  # type: ignore
```

Comment:
If mypy is complaining about these lines, I’d add ignores to `mypy.ini` so they apply for all imports of these packages, instead of having to remember to add `# type: ignore` every time they are imported. There are examples of how to ignore packages in the morphodict `mypy.ini`.

```suggestion
import pytest
from django.core.management import call_command
from django.shortcuts import reverse
```